cluster/gce/util.sh

```diff
@@ -708,7 +708,7 @@ function create-autoscaler() {
     local min_instances_per_mig=$(((${AUTOSCALER_MIN_NODES} + ${NUM_MIGS} - 1) / ${NUM_MIGS}))
     local last_min_instances=$((${AUTOSCALER_MIN_NODES} - (${NUM_MIGS} - 1) * ${min_instances_per_mig}))
 
-    for i in $(seq $((${NUM_MIGS} - 1))); do
+    for ((i=1; i<=$((${NUM_MIGS} - 1)); i++)); do
```

Comment:
nit: Isn't "i<$NUM_MIGS" equivalent and simpler?  Same in above loop on line 650.